rancher/installing-rancher/installing-server/basic-ssl-config/index.md

```diff
@@ -66,7 +66,48 @@ server {
 **Important Setting Notes:**
 
 * `rancher-server` is the name of your rancher server container. In other words, you must start your rancher server container with `--name=rancher-server` and your nginx container with `--link=rancher-server` for this exact configuration to work.
-* `<server>` can be any arbirtray name, but the same name should be used for both the http and https servers.
+* `<server>` can be any arbitrary name, but the same name should be used for both the http and https servers.
+
+
+## Apache Configuration
+
+Here is an Apache configuration. You'll need to launch an Apache container and do a docker link to provide DNS resolution for Rancher. 
+
+```
+<VirtualHost *:80>
+ServerName myhost.rancher.io
```

Comment:
Just so its clear, lets use <server_name> instead of myhost.rancher.io so the user knows to substitute it.